### DISCUSSION TITLE
Add to card is disabled

### DISCUSSION THREAD

**Discussion by amirseyedian** [bagisto/bagisto#10339] [Q&A] [answered]
### Terms

- [x] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

master branch

### Issue Description

facing the same issue as #6339 and configurable products not showing up, I think it's due to sql not updating.

### Preconditions

php 8.3
mysql 9.1.0 on docker

### Steps to reproduce

```
git clone https://github.com/bagisto/bagisto.git
composer install
php artisan bagisto:install
php artisan serv
```
add a configurable product
the add to cart button is greyed out and sometimes the products wont even show up in the product listings

**Comment by amit-webkul** [comment]
Hello @amirseyedian,

Thank you for reporting the issue.

I have reviewed it and found that the issue is not present in stable branch 2.2.

Regarding the current behavior: When an inactive product is added to a configurable or grouped product, the system skips the inactive product and uses the remaining product variants or group options instead.

Please let me know if you have any issues in the stable 2.2 branch.

Thanks !

**Comment by amirseyedian** [comment]
Hello @amit-webkul
Thank you for taking the time to review my issue.
No im not facing the problem when I rollback to 2.2 as I mentioned before the issue exists on the master branch
I was able to fix it by wiping and remigrating the database.

but now when I enable inventory management on the product description page, the `add to cart` button is disabled although the product is in stock.
I wasnt able to fully traceback the problem and believe there is a problem with updating the sql table simultaneously.
Im keen to hear your input regarding this issue or if anyone else has experienced this as well

**Comment by amit-webkul** [comment]
Hello @amirseyedian,

Regarding your query, if I understand correctly, you are referring to the product view page. Everything seems to be working fine on this page from our end.

If you could provide a short video demonstrating the issue, it would help us better understand and assist you in finding a solution.

Thanks & Regards,

**Comment by arturslogins** [comment]
I have the same issue 

Creating product - button add to card  is disabled 

I think this is problem, 
There is no record in this table. 

I added it manually, and button is active now 

![image](https://github.com/user-attachments/assets/1ea3312d-571d-4396-bfef-0150ae92f100)

**Comment by amirseyedian** [comment]
yes I can confirm that the fix @arturslogins mentioned works.
manually adding a row to `product_inventory indices` seems to fix the issue. but its very impractical to have to add a sql row for each products

**Comment by amit-webkul** [comment]
@amirseyedian @arturslogins 

Could you kindly confirm which version of Bagisto you have set up? This information will help us recheck and investigate the issue you are experiencing on your end.

As of now, our stable branch is version 2.2, which does not have this bug. Please confirm so we can assist you more effectively.

I’m more than happy to help you resolve this.

Looking forward to your response!

**Comment by amirseyedian** [comment]
No im not facing the problem when I rollback to 2.2 as I mentioned before the issue exists on the master branch

**Comment by arturslogins** [comment]
@amit-webkul 
I can confirm that the issue exists on the master branch

**Comment by amit-webkul** [comment, marked as answer]
@amirseyedian @arturslogins 

Actually, the thing is, in the master version, Bagisto is being made compatible with Laravel 11. To ensure proper functionality, after the installation, you need to run the following command:

`php artisan queue:listen`

This is necessary to process queued jobs in real-time and handle tasks seamlessly in the background.

If you have further queries, feel free to ask here.

Thanks...
